nsxt/data_source_nsxt_compute_manager_test.go

```diff
@@ -11,24 +11,44 @@ import (
 )
 
 func TestAccDataSourceNsxtComputeManager_basic(t *testing.T) {
-	ComputeManagerName := getComputeManagerName()
-	testResourceName := "data.nsxt_edge_cluster.test"
+	computeManagerName := getComputeManagerName()
+	testResourceName := "data.nsxt_compute_manager.test"
 
 	resource.ParallelTest(t, resource.TestCase{
 		PreCheck: func() {
 			testAccOnlyLocalManager(t)
-			testAccTestMP(t)
 			testAccPreCheck(t)
 			testAccEnvDefined(t, "NSXT_TEST_COMPUTE_MANAGER")
 		},
 		Providers: testAccProviders,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccNSXComputeManagerReadTemplate(ComputeManagerName),
+				Config: testAccNSXComputeManagerReadTemplate(computeManagerName),
 				Check: resource.ComposeTestCheckFunc(
-					resource.TestCheckResourceAttr(testResourceName, "display_name", ComputeManagerName),
+					resource.TestCheckResourceAttr(testResourceName, "display_name", computeManagerName),
+					resource.TestCheckResourceAttrSet(testResourceName, "id"),
+					resource.TestCheckResourceAttrSet(testResourceName, "server"),
+				),
+			},
+		},
+	})
+}
+
+func TestAccDataSourceNsxtComputeManager_single(t *testing.T) {
+	testResourceName := "data.nsxt_compute_manager.test"
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			testAccOnlyLocalManager(t)
```

Comment:
Not sure whether we need `testAccTestMP()` for these resources or not. Other than this LGTM

Reply:
I think we should rename `testAccTestMP` to `testAccTestDeprecated`. I think we should include the new fabric resources in the main test suite

Reply:
Thanks!